randovania/generator/reach_lib.py

```diff
@@ -156,5 +156,6 @@ def advance_to_with_reach_copy(base_reach: GeneratorReach, state: State) -> Gene
     potential_reach = copy.deepcopy(base_reach)
     potential_reach.advance_to(state)
     collect_all_safe_resources_in_reach(potential_reach)
+    if state.patches.configuration.consider_possible_unsafe_resources:
```

Comment:
I prefer to keep all generator parameters as part of `FillerConfiguration`, even if they come from the Preset.
(ideally GamePatches wouldn't have the configuration field...)

Reply:
bleh that's a little annoying since the reach API doesn't currently have access to a FillerConfiguration